core/constants-dev.yaml

```diff
@@ -35,8 +35,14 @@ vssMaxTTL: 6 # epochs
 vssMinTTL: 2 # epochs
 mpcSendInterval: 10 # must be less than (2 * k * slotDuration - networkDiameter)
 
-# Lightweight delegation
+# Delegation
 lightDlgConfirmationTimeout: 500
+# This value parameterizes size of dlg cache. There are two types of messages:
```

Comment:
No, this size is not in bytes, just size of the map. This comment explains that the bigger this value is, the bigger cache will be. However, I intentionally didn't put details here, because, as I said, I am almost sure nobody will keep this comment up-to-date, so I think the only source of truth should be in code.

But ok, I can also write that it's not size in bytes.